spec/schemas/flow_framework._common.yaml

```diff
@@ -0,0 +1,100 @@
+openapi: 3.1.0
+info:
+  title: Schemas of flow_framework._common category
+  description: Schemas of flow_framework._common category
+  version: 1.0.0
+paths: {}
+components:
+  schemas:
+    FlowFrameworkException:
+      content:
+        application/json:
+          type: object
+          properties:
+            message:
+              type: string
+              # example: "This API is disabled. To enable it, set [flow_framework.enabled] to true."
```

Comment:
For my own education, is this a comment to aid review or does this have any meaning in the spec?